cloudslang_best_practices.rst

```diff
@@ -59,27 +60,44 @@ All CloundSlang flows and operations should begin with a documentation
 block that describes the flow or operation, and lists the inputs,
 outputs and results.
 
-Structure
----------
+A flow or operation's documentation may be viewed from the CLI using the
+:ref:`inspect <inspect_a_flow_or_operation>` command.
 
-The structure and spacing of the comments are as in the example below:
+-  Documentation blocks begin with a line containing ``#!!`` and nothing else.
+-  Documentation blocks end with a line containing ``#!!#`` and nothing else.
+-  Each line of the documentation begins with ``#!``.
+-  Lines in the documentation block that do not begin with ``#!`` will not be
+   considered as part of the documentation and will not display when the file is
+   inspected.
+-  The ``@description`` tag is the only mandatory tag.
+-  The other possible tags are:
+
+     - ``@prerequisites``
+     - ``@input <input_name>``
+     - ``@output <output_name>``
+     - ``@result <result_name>``
 
 ::
 
     ####################################################
-    # Does something fantastic.
-    #
-    # Prerequisites: some Python module
-    #
-    # Inputs:
-    #   - input1 - first input
-    #   - input2 - optional - port for something - Default: 8080
-    #   - list - a list of things - Format: comma delimited list of things
-    # Outputs:
-    #   - output1 - some sort of message
-    # Results:
-    #   - SUCCESS - everything went well
-    #   - FAILURE - otherwise
+    #!!
+    #! @description: Does something fantastic.
+    #!
+    #! @prerequisites: Some Python module.
+    #!
+    #! @input input_1: first input
+    #! @input input_2: second input
+    #!       default: true
```

Comment:
also maybe align the "default: true" with "second input" but this is not a big deal